es-design-system/pages/molecules/es-progress-circle.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <div>
+        <p class="pb-200">
+            Extended from <b-link href="https://github.com/EnergySage/es-cdgm/blob/main/frontend/components/CircularProgress.vue">
+                <code>es-cdgm</code> codebase
+            </b-link>
+        </p>
+        <h2>
+            Various Heights
+        </h2>
+        <div class="d-flex flex-nowrap">
+            <div
+                v-for="height in heightRange"
+                :key="height"
+                class="d-flex flex-nowrap align-items-center mr-100">
+                <p class="mr-2">
+                    Height: {{ height }}px
+                </p>
+                <es-progress-circle
+                    :value="height"
+                    :height="`${height}px`" />
+            </div>
+        </div>
+        <h2 class="pt-200">
```

Comment:
It looks like this currently there aren't any animations for the progress circle. I took a pass on [how we might implement something like that](https://codesandbox.io/p/sandbox/peaceful-pateu-8m7c4p?file=%2FREADME.md), but I think that's probably outside the scope of this PR. Certainly something we can add at a later time.

Reply:
Just after a glance I think it is out of scope -- logged a ticket, number is 158